src/main/java/appeng/items/tools/powered/powersink/IC2.java

```diff
@@ -19,7 +19,7 @@
 package appeng.items.tools.powered.powersink;
 
 
-//import com.google.common.base.Optional;
+//import java.util.Optional;
```

Comment:
Unrelated, but these classes should really go away and only be replaced by item capabilities.